### PR TITLE
Add a section about Passim in the top-level README file

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,18 @@ This will download and apply all updates for your system.
...
+
+If the [Passim](https://github.com/hughsie/passim/blob/main/README.md) project is also installed
+and enabled, fwupd will re-publish the downloaded metadata file to be served on `0.0.0.0:27500`
+by default.
+
+Other clients on the same network can make use of this via mDNS/LLMNR to reduce network bandwidth
+to configured remotes.
+
+To disable this functionality either set `P2pPolicy=none` in `/etc/fwupd/daemon.conf`, uninstall
+the passim package or use `systemctl mask passim.service` on the terminal.
```
Comment (superm1) [inline README.md:99]: Generally this looks good, but I unfortunately don't think it works.  You can't uninstall the package and have it run.

https://github.com/fwupd/fwupd/issues/7173

Comment (hughsie) [reply README.md:99]: Ohh, that's just Arch -- on Fedora we split out the library from the daemon:

    passim-libs-0.1.7-1.fc40.x86_64
    passim-0.1.7-1.fc40.x86_64
    passim-devel-0.1.7-1.fc40.x86_64

Comment (superm1) [reply README.md:99]: Ahh I see.

**Review by superm1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hughsie) [reply README.md:99]: Ohh it's a downstream split -- upstream just blits files to a destdir -- it's up to the packaging system to split them up into -libs, -devel etc.

**Pull request merged**